auto_backup/tests/test_db_backup.py

```diff
@@ -151,7 +151,7 @@ def test_action_backup_sftp_mkdirs(self):
                 with patch("%s.cleanup" % class_name, new_callable=PropertyMock):
                     conn = rec_id.sftp_connection().__enter__()
                     rec_id.action_backup()
-                    conn.makedirs.assert_called_once_with(rec_id.folder)
+                    conn.makedirs.assert_called_once_with(rec_id.folder, exist_ok=True)
```

Comment:
```suggestion
                    conn.makedirs.assert_called_once_with(rec_id.folder)
```
see previous comment.
Could you please add a test to check for the change to local backups instead?